tokio-trace-serde/Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "tokio-trace-serde"
+version = "0.1.0"
+authors = ["J Haigh <debugsteven@gmail.com>"]
+edition = "2018"
+
+[dependencies]
+serde = "1.0.90"
+tokio-trace = { git = "https://github.com/tokio-rs/tokio" }
```

Comment:
Also, it occurs to me that this crate seems to only use the types defined in `tokio-trace-core`, rather than `tokio-trace`, so we should probably depend on the `tokio-trace-core` crate instead. That will make it easier to use this crate in subscriber implementations that may not depend on the instrumentation-focused APIs in `tokio-trace`.